licenses/models.py

```diff
@@ -342,4 +342,30 @@ class TranslationBranch(models.Model):
     last_transifex_update = models.DateTimeField(
         "Time when last updated on Transifex.", null=True, blank=True,
     )
-    complete = models.BooleanField("Only one incomplete per branch", default=False,)
+    complete = models.BooleanField(default=False)
+
+    class Meta:
+        verbose_name_plural = "translation branches"
+
+    def __str__(self):
+        return f"Translation branch {self.branch_name}. {'Complete' if self.complete else 'In progress'}."
+
+    @property
+    def stats(self):
+        untranslated_messages = 0
+        translated_messages = 0
+        for code in self.legalcodes.all():
+            pofile = code.get_pofile()
+            untranslated_messages += len(pofile.untranslated_entries())
+            translated_messages += len(pofile.translated_entries())
+        total_messages = untranslated_messages + translated_messages
+        if total_messages:
+            percent = int(translated_messages * 100 / float(total_messages))
```

Comment:
I think `len()` returns an `int`. I think we do not need to convert `total_messages` into a `float`.

Reply:
I just copied this from ```polib.POFile.percent_translated``` without thinking too hard about it.  I guess I'd better think about it... :-)

The intention is probably to force a float division rather than an integer division for greater accuracy before truncating the result. Given that the numerator is already multiplied by 100, I doubt it makes any difference in practice, but I don't think it hurts anything either.